src/_scss/pages/recipient/_contentWrap.scss

```diff
@@ -80,6 +80,23 @@
 				@include media($tablet-screen) {
 					@include flex(1 1 50%);
 				}
+
+				.recipient-section__award-button {
+					@include display(flex);
+					@include flex(0 0 auto);
+					@include align-items(center);
+					@include button-unstyled;
+					color: $color-primary;
+					margin-top: rem(16);
+					font-size: rem(14);
+
+					&:hover {
+						color: $color-primary-darker;
+						svg {
+							fill: $color-primary-darker;
+						}
```

Comment:
the link includes a lot of white area.  when i hover over the white area, the link is active. also shows when the link has focus.  The first screenshot I'm hovering over the area on the right of the text, not the text itself - 
<img width="1121" alt="Screen Shot 2022-08-05 at 10 42 39 AM" src="https://user-images.githubusercontent.com/121010/183101204-89af2323-f298-478d-8f1d-2e7bf3656a33.png">

<img width="1121" alt="Screen Shot 2022-08-05 at 10 40 52 AM" src="https://user-images.githubusercontent.com/121010/183101258-703db12d-9cc6-4d7b-aff1-6918bbf87a71.png">

Reply:
good catch, fixed width